interview-questions.md

```diff
@@ -0,0 +1,102 @@
+# ASSESSMENT 2: Interview Practice Questions
+
+Answer the following questions.
+
+First, without external resources. Challenge yourself to answer from memory as if you were in a job interview.
+
+Then, research the question to expand on your answer. Even if you feel you have answered the question completely, there is always something more to learn. Write your researched answer in your OWN WORDS.
+
+1. What is the difference between a parameter and an argument?
+
+  Your answer: Overall, a parameter and an argument refer to the same data. When creating a function, a parameter will be the naming convention that acts as the value that is passed in the function. When invoking the function, data may be passed into a function as an argument. The value of the parameter is based on what is passed into the function as an argument. 
+
+  Researched answer: A parameter is a placeholder variable that is definied INSIDE the the parentheses of the function expression  where it may be leveraged/utilized anywhere within the function logic. An argument is the actual value that is passed to the function when the function is invoked. The parameter is then assigned this value.
+
+  - Reference: [Functions Overview](https://github.com/learn-academy-2022-charlie/Syllabus/blob/main/javascript/functions.md)
```

Comment:
Correct the parameter is a local variable that is scoped to the function.

Reply:
Ahhh okay yes, I thought about referring a parameter as a variable but was unsure if that was the correct terminology. Thank you!